full-check.sh

```diff
@@ -0,0 +1,19 @@
+printf "=================== DJANGO CONFIGURATION CHECKS ====================\n"
+python manage.py check
+printf "\n"
+
+printf "=============================== LINT ===============================\n"
+./lint.sh
+printf "\n"
+
+printf "========================= MYPY STATIC TYPE CHECKER =================\n"
+mypy --config-file=mypy.ini time_monkey/
+mypy --config-file=mypy.ini users/
+mypy --config-file=mypy.ini employees/
+mypy --config-file=mypy.ini managers/
```

Comment:
None of them exists now. Please delete them

Reply:
Done